File: frontend/src/features/actividades/rectificatoria_analisis_derivacion_reparo_postventa/models/rectificatoria_analisis_derivacion_reparo_postventa.ts
export interface RectificatoriaAnalisisDerivacionReparoPostventa {
  id_rectificatoria_analisis_derivacion_reparo_postventa: number;
  id_expediente: number;
  enviar_reparo_a?: boolean | null;
  observaciones?: string | null;

  is_active: boolean;
  row_status: boolean;
  created_by: number;
  created_date: string;
  modified_by?: number | null;
  modified_date?: string | null;
}

File: backend/Multibanca.Domain.Models/Multibanca/rectificatoria_analisis_derivacion_reparo_postventa.cs
using Common.Domain.Models;
using System;

namespace Multibanca.Domain.Models.Multibanca
{
    public class rectificatoria_analisis_derivacion_reparo_postventa : base_auditoria
    {
        public int id_rectificatoria_analisis_derivacion_reparo_postventa { get; set; }
        public long id_expediente { get; set; }
        public string? enviar_reparo_a { get; set; }
        public string? observaciones { get; set; }
    }
}
